### PR TITLE
Fix contradiction in “Using Minikube to Create a Cluster”

### PR DESCRIPTION
**Pull request by sravyasambaturu**
**Line 73** -- Each node has a Kubelet, which is an agent for managing the node and communicating with the Kubernetes control plane 
**Line 85** -- The nodes communicate with the control plane using the Kubernetes API which the control plane exposes.

Hello, 
I feel like these two kind of contradict.. the nodes has kubelet as an agent that helps in communicating with the kubernetes API that exists in control plane...

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h2 style="color: #3771e3;">Cluster Diagram</h2>
 
         <div class="row">
             <div class="col-md-8">
-                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>The nodes communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
+                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>The Kubelet communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:85]: How about:
```suggestion
                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>The kubelet on each node communicates with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
```

Comment (sravyasambaturu) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:85]: right, makes more sense.

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this
/assign @sftim

**Comment by sftim** [comment]
@sravyasambaturu what are your plans for this pull request? You could apply the change that I suggested.

**Review by sravyasambaturu — COMMENTED** [review]
updated the changes /approve

**Comment by natalisucks** [comment]
@sravyasambaturu Hey there 👋 You'll need to push an empty commit to retrigger the checks on this PR, as they're currently failing.

Once that's done, we can look to getting this merged – the change looks fine to me.

**Comment by sftim** [comment]
/retitle Fix contradiction in “Using Minikube to Create a Cluster”

**Comment by sftim** [comment]
Hi @sravyasambaturu 

This change doesn't build OK. Would you like some help in fixing it so that it does?

If you prefer to close this pull request and start a new one that makes similar changes, you can do that.

**Comment by sravyasambaturu** [comment]
@sftim as per your suggestion created a new PR https://github.com/kubernetes/website/pull/41946 
 So closing this one.

**Pull request closed without merging**